modules/understanding-agent-install.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent/installing-with-agent.adoc
+
+:_content-type: CONCEPT
+[id="understanding-agent-install_{context}"]
+= Understanding Agent-based installer
+As a {product-title} user, you can leverage all the advantages of the Assisted Installer hosted service in disconnected environments.
+
+The Agent installation comprises a bootable ISO that contains the Assisted discovery agent and the Assisted service; both are required to perform the first cluster installation.
+
+The `openshift-install agent create image` subcommand generates an ephemeral ISO based on the inputs that you provide. You can choose to provide inputs through the following manifests:
+
+Preferred:
+
+* install-config.yaml
+* agent-config.yaml
+
+or
+
+Optional: ZTP manifests
```

Comment:
This [explanation of ZTP manifests](http://file.del.redhat.com/asakthiv/agent-install-new/scalability_and_performance/ztp-deploying-disconnected.html#ztp-ztp-custom-resources_ztp-deploying-disconnected) was buried in the link provided but can we link more directly to it somewhere?

Reply:
@lranjbar ptal http://file.del.redhat.com/asakthiv/agent-install-new/installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#sample-ztp-custom-resources_installing-with-agent-based-installer  . I have added a link in the **Additional Resources** section. Since in this section , we talk more in details about the ZTP manifests.